lib/screens/user_registration/user_registration.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:flutter/material.dart';
+import 'package:teesco/screens/login/widgets/password_field.dart';
+import 'package:teesco/screens/user_registration/widgets/email_input.dart';
+import 'package:teesco/screens/user_registration/widgets/institute_name_input.dart';
+import 'package:teesco/screens/user_registration/widgets/phone_number_field.dart';
+import 'package:teesco/screens/user_registration/widgets/confirm_password_field.dart';
+
+class UserRegistration extends StatelessWidget {
+  TextEditingController userNameController;
```

Comment:
Did this code work?
As i see, you have declared all TextEditingControllers but haven't initialized them.
Also, stateless widgets are not supposed to declare non-final variables. If you do, it raises a warning:
`This class (or a class that this class inherits from) is marked as '@immutable', but one or more of its instance fields aren't final:`
```suggestion
  final TextEditingController userNameController = TextEditingController();
```
refer the LoginScreen widget.

Reply:
Did that. Can I have related docs over the channel